server/container_create.go

```diff
@@ -286,119 +285,6 @@ func generateUserString(username, imageUser string, uid *types.Int64Value) strin
 	return userstr
 }
 
-// setupCapabilities sets process.capabilities in the OCI runtime config.
-func setupCapabilities(specgen *generate.Generator, caps *types.Capability, defaultCaps capabilities.Capabilities) error {
-	// Remove all ambient capabilities. Kubernetes is not yet ambient capabilities aware
-	// and pods expect that switching to a non-root user results in the capabilities being
-	// dropped. This should be revisited in the future.
-	specgen.Config.Process.Capabilities.Ambient = []string{}
```

Comment:
This line is missing in the moved function.

Reply:
it's done in ClearProcessCapabilities(). as long as it's not updated later we're good